docs/rbd-nbd.md

```diff
@@ -38,6 +38,7 @@ Rbd-nbd support status: **Alpha**
 | ---------------------------------------- | -------------- | ------------------ | -------------------- | ---------------- | ------------------ |
 | Creating and deleting snapshot           | Alpha          | >= v3.4.0          | Pacific (>=16.0.0)   | >= v1.0.0        | >= v1.17.0         |
 | Creating and deleting clones             | Alpha          | >= v3.4.0          | Pacific (>=16.0.0)   | >= v1.0.0        | >= v1.17.0         |
+| Expand volumes                           | Alpha          | >= v3.4.0          | Pacific (>=16.0.0)   | >= v1.1.0        | >= v1.15.0         |
```

Comment:
I think its better to track the features which are out of CSI spec, in  a section at common readme for a single place to refer. For ex: rbd nbd , encryption, mirroring ..etc are possible items which can be specified.

Reply:
For rbd-nbd there is `Please refer rbd nbd mounter for its support details.` in the top readme file

Reply:
@pkalever what I was actually saying here is ,  instead of keeping the support matrix of nbd here in this doc, rather have one section in common readme which we were planning before the release for features which are outside of CSI spec and keep it. Any way , it can be a folllow up PR too , I will rearrange the same

Reply:
I see. As far as I can remember, we didn't want to make the matrix at README more clumsy. Because nbd should be under rbd plugin again and it needs subsections for krbd and nbd both carrying snapshots/clones/creates/encryption... kind of features. So we thought of keeping it separate to avoid complexity in understanding. Thanks!